CHANGELOG.next.toml

```diff
@@ -235,3 +235,45 @@ message = "Support for constraint traits on member shapes (constraint trait prec
 references = ["smithy-rs#1969"]
 meta = { "breaking" = false, "tada" = true, "bug" = false, "target" = "server" }
 author = "drganjoo"
+
+[[smithy-rs]]
+message = """
+Add support for the `awsQueryCompatible` trait. This allows services to continue supporting a custom error code (via the `awsQueryError` trait) when the services migrate their protocol from `awsQuery` to `awsJson1_0` annotated with `awsQueryCompatible`. After the migration, services will include an additional header `x-amzn-query-error` in their responses whose value is in the form of `<error code>;<error type>`. An example response looks something like
```

Comment:
Can you put everything after the first sentence into an expand/collapse?

Reply:
Good suggestion! Addressed in a8386bd.